packages/pretty-format/src/index.js

```diff
@@ -317,16 +317,16 @@ function printPlugin(val, indent, prevIndent, spacing, edgeSpacing, refs, maxDep
 }
 
 function print(val, indent, prevIndent, spacing, edgeSpacing, refs, maxDepth, currentDepth, plugins, min, callToJSON, printFunctionName, escapeRegex, colors) {
-  const basic = printBasicValue(val, printFunctionName, escapeRegex);
-  if (basic) {
-    return basic;
-  }
-
   const plugin = printPlugin(val, indent, prevIndent, spacing, edgeSpacing, refs, maxDepth, currentDepth, plugins, min, callToJSON, printFunctionName, escapeRegex, colors);
   if (plugin) {
     return plugin;
   }
 
+  const basic = printBasicValue(val, printFunctionName, escapeRegex);
```

Comment:
That's not the only way to specify a jest config, unfortunately. Would you mind divorcing the two changes you are making in this diff and sending a separate PR for the filepath thing? Thanks!